cmd/config-sync/config_sync.go

```diff
@@ -232,6 +232,8 @@ func syncRouterConfig(agent *qdr.Agent, desired *qdr.RouterConfig, c *ConfigSync
 	}
 
 	differences := qdr.ConnectorsDifference(actual, desired)
+	differences = filterInternalRouterConnectors(differences, agent, SHARED_TLS_DIRECTORY)
```

Comment:
Minor point, feel free to ignore. To me, it would be clearer that it is only filtering the deleted connectors if only the deleted list was passed in, e.g. something like:

```differences.Deleted = filterInternalRouterConnectors(differences.Deleted, agent, SHARED_TLS_DIRECTORY)```

Reply:
I agree, makes the code much clearer

Reply:
I'd possibly even consider putting the filtering directly into the qdr.ConnectorsDifference to save having to lookup the connector by name again.

Reply:
You are right! I will work on that